config/config.go

```diff
@@ -22,13 +22,20 @@ type GitHubURLs struct {
 
 // Repo represents any kind of repo (github, gitlab, etc)
 type Repo struct {
-	Owner string `yaml:",omitempty"`
-	Name  string `yaml:",omitempty"`
+	Owner  string       `yaml:",omitempty"`
+	Name   string       `yaml:",omitempty"`
+	Author CommitAuthor `yaml:",omitempty"`
```

Comment:
I would put this in the brew section, since repo can be used in the release as well and it does not apply there..

Reply:
Agree. Was thinking that this is easier to understand but guess we can fix with a comment in the docs.